app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,24 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Pairin - Find pair programming partner or coding buddy quick and easy</title>
+    <title inertia>Pairin - Find pair programming partner or coding buddy quick and easy</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <meta name="description" content="Discover the perfect pair programming partner or coding buddy with Pairin.dev. Enhance your coding skills and collaborate in real-time with developers around the world." />
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= vite_stylesheet_tag "application" %>
+    <%= vite_react_refresh_tag %>
+    <%= vite_client_tag %>
+    <%= vite_javascript_tag "inertia" %>
+    <%= inertia_ssr_head %>
+    <%= vite_stylesheet_tag 'application', data: { 'turbo-track': 'reload' } %>
```

Comment:
This seems like a duplicate of line 10?(though I guess we need to keep `date: {...` part)